### PR TITLE
Fixed typos and minor edits

### PR DESCRIPTION
**Pull request by mcmimik**
Fixed two typos; removed extra void's parentheses and redundant semicolon; fixed setting css values with strings (see https://stackoverflow.com/a/55745605/3386279).

### CODE REVIEW

**Comment by mcmimik** [comment]
> `void(0)` parentheses are required for older browsers.

Could you please name that browsers and its versions? I like to catch errors in old browser too, but it seems there is no such browsers (because `void` is an operator, not a function): https://stackoverflow.com/questions/13656453/many-people-write-javascriptvoid0-instead-of-javascriptvoid-0-in-hrefs

**Comment by bluesmoon** [comment]
I honestly cannot remember. When I wrote this, IE6 still had market share. I'm ok with changing it if you can be sure it works on at leat IE8. You'll still need the semi colon at the end as that's the programming style we use.

**Comment by mcmimik** [comment]
I checked, it works:

image

File: README.md [hunk 1]
```diff
@@ -260,9 +260,9 @@ For boomerang, the asynchronous loader snippet you'll use is:
 
         // Hide the iframe
         iframeStyle = (iframe.frameElement || iframe).style;
-        iframeStyle.width = 0;
-        iframeStyle.height = 0;
-        iframeStyle.border = 0;
+        iframeStyle.width = "0";
```
Comment (nicjansma) [inline README.md]: I don't think the `"`s are necessary if you're setting to `0` specifically.

Comment (mcmimik) [reply README.md]: Fixed.

**Pull request merged**